### ISSUE TITLE
-moz-win-borderless-glass is broken in areas, causes a black box in front of elements drawn by this code

### ISSUE THREAD

**Issue by DAPOPOBEFASTONYOAZZ** [NetworkNeighborhood/Marble-legacy#5]
I have CSS code that revives a classic extension, GlassMyFox, which made many elements of the browser drawn by Windows Aero, like context menus, library panes, bookmark sidebar and more.

I had been using r3dfox before, but now trying out Marble, this build breaks -moz-win-borderless-glass in several areas.

Before, what looked like this on r3dfox:
![image](https://github.com/user-attachments/assets/4d3f9b87-be07-4646-a912-12e54c1f2b7c)

Now looks like this on Marble:
![image 2](https://github.com/user-attachments/assets/c67010a3-27b9-4d66-aa99-047a1a659240)

You can tell that in Marble in the current state, in the context menu, it is still trying to draw its region using Windows Aero, as is evident by its translucent and blurry 1px - 2px border. But something about Marble breaks it. I thought this was something about my userChrome, but no matter what I do, I cannot get the black box to go away.

Also, other areas, like the downloads drop down, or the tab preview pop-up are completely transparent, whereas these should have also been drawn by Windows Aero when using -moz-win-borderless-glass. Prior in r3dfox, they had the same issue of the black box but still being drawn by Windows Aero underneath, but now these are completely transparent and seem to ignore -moz-win-borderless-glass, which make it quite impossible to read.
![image 3](https://github.com/user-attachments/assets/3ce4f1d8-5234-44a3-89e2-ce564f775a45)

If you need me to share my userChrome files, I will attach them for you for debugging.

Edit: Using Windows 10 with OpenGlass.

**Comment by Erizur** [comment]
Seems like it's an issue where -moz-borderless-glass (in this case, DWM extended frames) isn't applying to all windows except for the main one. Could you test with Marble 132 perchance?

**Comment by DAPOPOBEFASTONYOAZZ** [comment]
Yes, my userChrome code works just fine in Marble 132 and appears as such:
![Untitled](https://github.com/user-attachments/assets/3d4ca54c-0d6a-4343-8c05-550c5b4ded41)

**Comment by Erizur** [comment]
Since Firefox 133 added some Mica (Windows 11 additions, strange things related to glass and titlebar related stuff have been occuring on the latest version. I will have a look at it soon.

**Comment by Erizur** [comment]
Seems like I missed out on some code that restored the css property when merging the new 133 changes. I am not updating Marble 133 anymore, but I will certainly have this fixed for the ESR branch.

**Comment by Erizur** [comment]
Fixed as of Marble G1.
